### PR TITLE
create projects page

### PR DESCRIPTION
**Pull request by Jakeclearwater**
new projects landing page, waiting for dani to add firebase functionality to it

### CODE REVIEW

File: src/components/content/ProjectsContent.jsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    {
+      name: "Project 3",
+      creationDate: "2022-01-03",
+    },
+  ];
+
+  return (
+    <div className={classes.body}>
+      <h1>Projects</h1>
+      <Link to="/create-project" className={classes.button}>
```
Comment (Dani-op) [inline src/components/content/ProjectsContent.jsx:28]: Hi Jake,
The code appears to be in good shape, but there's an issue with it causing the site to crash when clicking on "Create Project," which is less than ideal. After debugging, I discovered that the problem lies with the "Link" tag used for "Create Project." To resolve this, please replace the "Link" tag with a button and remove the "href" attribute ("/create-project"). I'll provide the necessary changes below. Thank you.
chrome_ny8SIz4ac2

File path: src/components/content/ProjectsContent.jsx

From this :
image
To this:
image

**Review by Dani-op — CHANGES_REQUESTED** [review]
Hi Jake,
I am happy to review the PR again once the requested changes are made.
Thanks.

**Comment by Jakeclearwater** [comment]
done @Dani-op thank you for your feedback on the issue

**Review by Dani-op — APPROVED** [review]
Thank you @Jakeclearwater for making the necessary changes! Your quick response and action are greatly appreciated. The site should now run smoothly.

**Pull request merged**